ci/keystone/keystone.sh

```diff
@@ -60,7 +63,7 @@ case "$1" in
     pid=$!
     # Doing this after launching keystone will give it plenty of time to get
     # started without adding any wasteful calls to sleep:
-    pip install python-openstackclient
+    pip install python-openstackclient==3.2.0
```

Comment:
What's the reason for pinning this to `3.2.0`? Is there a more permissive constraint that "should" work? Either way, please document.